sdk/storage/storage-queue/package.json

```diff
@@ -43,7 +43,9 @@
     "unit-test:browser": "cross-env TEST_MODE=playback npm run integration-test:browser",
     "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "emulator-tests": "cross-env STORAGE_CONNECTION_STRING=UseDevelopmentStorage=true && npm run test:node"
+    "emulator-tests": "cross-env STORAGE_CONNECTION_STRING=UseDevelopmentStorage=true && npm run test:node",
+    "coverage-integration-test:node": "nyc npm run integration-test:node",
```

Comment:
I assume the code coverage report look similar before and after this change?

Reply:
The code coverage report looks the same as long as, after this change, you do `coverage-integration-test:node` to get the coverage 🌞